packages/web/src/js/utils/__tests__/Config.test.ts

```diff
@@ -0,0 +1,179 @@
+import { clearFixture, getFixture } from '../../../../tests/helpers/fixture';
+import Config from '../Config';
+
+class DummyConfigClass extends Config {
+  static get NAME() {
+    return 'dummy';
+  }
+}
+
+describe('Config', () => {
+  let fixtureEl: Element;
+  const name = 'dummy';
+
+  beforeAll(() => {
+    fixtureEl = getFixture();
+  });
+
+  afterEach(() => {
+    clearFixture();
+  });
+
+  describe('NAME', () => {
+    it('should return plugin NAME', () => {
+      expect(DummyConfigClass.NAME).toEqual(name);
+    });
+  });
+
+  describe('DefaultType', () => {
+    it('should return plugin default type', () => {
+      expect(DummyConfigClass.DefaultType).toEqual({});
+    });
+  });
+
+  describe('Default', () => {
+    it('should return plugin defaults', () => {
+      expect(DummyConfigClass.Default).toEqual({});
+    });
+  });
+
+  describe('mergeConfigObj', () => {
+    it("should parse element's data attributes and merge it with default config. Element's data attributes must excel Defaults", () => {
+      fixtureEl.innerHTML =
+        '<div id="test" data-spirit-test-bool="false" data-spirit-test-int="8" data-spirit-test-string1="bar"></div>';
+
+      jest.spyOn(DummyConfigClass, 'Default', 'get').mockReturnValue({
+        testBool: true,
+        testString: 'foo',
+        testString1: 'foo',
+        testInt: 7,
+      });
+
+      const configResult = DummyConfigClass.mergeConfigObj({}, fixtureEl.querySelector('#test'));
+
+      expect(configResult?.testBool).toBe(false);
+      // expect(configResult?.testString).toBe('foo');
```

Comment:
Do we want to keep this?

Reply:
I am trying to figure out the mocking of the static properties.

Reply:
Fixed.